### PR TITLE
Add diagnostic sensor instructions to Total Connect

### PR DESCRIPTION
**Pull request by austinmroczek**
## Proposed change
Add instructions about the diagnostic sensors available for TotalConnect.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/73152
- Link to parent pull request in the Brands repository: n/a
- This PR fixes or closes issue: fixes # n/a

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/totalconnect.markdown [hunk 1]
```diff
@@ -93,3 +93,11 @@ The `triggered` state also provides a state attribute called `triggered_source`
...
 
 The Total Connect API has limited zone type information. Home Assistant device class `door` is assigned to Total Connect entry/exit, perimeter, and motion zones; along with most alarm panel buttons. The sensor will appear as `True` if the zone is opened (either fault or triggered in Total Connect) and `False` if closed. Device class `smoke` is assigned to Total Connect smoke detectors and alarm panel buttons set to a "Fire No Verification" response type. The sensor will appear as `True` if smoke is detected or the respective button is pressed.  Device class `gas` is assigned to Total Connect carbon monoxide detectors. The sensor will appear as `True` if carbon monoxide is detected.
+
+### Diagnostic Sensors
+
+The following diagnostic sensors will be generated, but hidden by default:
```
Comment (gjohansson-ST) [inline source/_integrations/totalconnect.markdown]:
```suggestion
The following diagnostic sensors are available:
```
Not sure what you mean by hidden?

File: source/_integrations/totalconnect.markdown [hunk 2]
```diff
@@ -93,3 +93,11 @@ The `triggered` state also provides a state attribute called `triggered_source`
...
+
+- Low Battery for Zones and Alarm Panels will be `True` if the battery is low
+- Tamper for Zones and Alarm Panels will be `True` if in a tamper state
+- Power for Alarm Panels will be `True` if power is connected, or `False` if running on backup battery.
```
Comment (gjohansson-ST) [inline source/_integrations/totalconnect.markdown]: Binary sensors are on or off

**Review by gjohansson-ST — DISMISSED** [review]

File: source/_integrations/totalconnect.markdown (hunk 2 above)
Comment (austinmroczek) [reply source/_integrations/totalconnect.markdown]: Fixed

File: source/_integrations/totalconnect.markdown [hunk 3]
```diff
@@ -93,3 +93,11 @@ The `triggered` state also provides a state attribute called `triggered_source`
...
 
 The Total Connect API has limited zone type information. Home Assistant device class `door` is assigned to Total Connect entry/exit, perimeter, and motion zones; along with most alarm panel buttons. The sensor will appear as `True` if the zone is opened (either fault or triggered in Total Connect) and `False` if closed. Device class `smoke` is assigned to Total Connect smoke detectors and alarm panel buttons set to a "Fire No Verification" response type. The sensor will appear as `True` if smoke is detected or the respective button is pressed.  Device class `gas` is assigned to Total Connect carbon monoxide detectors. The sensor will appear as `True` if carbon monoxide is detected.
+
+### Diagnostic Sensors
+
+The following diagnostic sensors are available:
+
+- Low Battery for Zones and Alarm Panels will be `On` if the battery is low
+- Tamper for Zones and Alarm Panels will be `On` if in a tamper state
+- Power for Alarm Panels will be `On` if main power is connected, or `Off` if running on backup battery.
```
Comment (frenck) [inline source/_integrations/totalconnect.markdown]:
```suggestion
- Low Battery for Zones and Alarm Panels will be `On` if the battery is low.
- Tamper for Zones and Alarm Panels will be `On` if in a tampered state.
- Power for Alarm Panels will be `On` if main power is connected or `Off` if running on the backup battery.
```

**Review by frenck — APPROVED** [review]
Thanks, @austinmroczek 👍 

../Frenck

**Pull request merged**